gitops/pipelines/pipelines-controller/base/deployment.yaml

```diff
@@ -39,6 +39,8 @@ spec:
           env:
             - name: KUBECONFIG
               value: /workspace/kubeconfig/admin.kubeconfig
+            - name: KUBERNETES_MIN_VERSION
```

Comment:
Is this still needed? I think the kcp team has fixed it.

Reply:
Env var removed